server/routes/viewingLicences/handlers/printLicence.ts

```diff
@@ -0,0 +1,81 @@
+import { Request, Response } from 'express'
+import logger from '../../../../logger'
+import config from '../../../config'
+import PrisonerService from '../../../services/prisonerService'
+
+const pdfHeaderFooterStyle =
+  'font-family: Arial; ' +
+  'font-size: 10px; ' +
+  'font-weight: normal; ' +
+  'width: 100%; ' +
+  'height: 35px; ' +
+  'text-align: center; ' +
+  'padding: 20px;'
+
+export default class PrintLicenceRoutes {
+  constructor(private readonly prisonerService: PrisonerService) {}
+
+  preview = async (req: Request, res: Response): Promise<void> => {
+    const { username } = res.locals.user
+    const { licence } = res.locals
+    const htmlPrint = true
+    logger.info(`HTML preview licence ID [${licence.id}] type [${licence.typeCode}] by user [${username}]`)
+    res.render(`pages/licence/${licence.typeCode}`, { licence, htmlPrint })
```

Comment:
Dont need to pass licence into context here, its already in global context `res.locals`

Reply:
Yes, you're right - a hangover from when this was a POST route. I will sort this.

Reply:
Not needed.